.env

```diff
@@ -0,0 +1,2 @@
+metadata_preview_js_url="https://access.sit.earthdata.nasa.gov/plugin/metadata-preview.0.0.38.js"
```

Comment:
I'm wondering if we should check this in?   I guess it will be overwritten by bamboo.   If you keep, wonder if we should put a production version in there?

Reply:
Correct, this should not be checked in. Will remove it.